Paintroid/src/main/java/org/catrobat/paintroid/dialog/InfoDialog.kt

```diff
@@ -29,39 +29,43 @@ import androidx.core.os.bundleOf
 import org.catrobat.paintroid.R
 
 class InfoDialog : AppCompatDialogFragment(), DialogInterface.OnClickListener {
-	override fun onCreateDialog(savedInstanceState: Bundle?): Dialog {
-		val builder = AlertDialog.Builder(requireContext(), R.style.PocketPaintAlertDialog)
-		arguments?.let {
-			builder.setTitle(it.getInt(TITLE_RESOURCE_KEY))
-					.setIcon(it.getInt(DRAWABLE_RESOURCE_KEY))
-					.setMessage(it.getInt(MESSAGE_RESOURCE_KEY))
-		}
-		builder.setPositiveButton(android.R.string.ok, this)
-		return builder.create()
-	}
+    override fun onCreateDialog(savedInstanceState: Bundle?): Dialog {
+        val builder = AlertDialog.Builder(requireContext(), R.style.PocketPaintAlertDialog)
+        arguments?.let {
```

Comment:
I feel like whenever we use the implicit lambda parameter, it should always be renamed to a more descriptive name. Is there perhaps a way to enforce this with detekt?